src/Middleware/ThrottleMiddleware.php

```diff
@@ -186,6 +186,7 @@ protected function _rateLimit(ThrottleInfo $throttle): RateLimitInfo
         $event = $this->dispatchEvent(self::EVENT_BEFORE_CACHE_SET, [
             'rateLimit' => $rateLimit,
             'ttl' => $ttl,
+            'throttleInfo' => clone$throttle,
```

Comment:
Why is the test suite not failing in spite of this? :fearful:

Reply:
https://www.php.net/manual/en/language.oop5.cloning.php
`clone($obj)` is replaced by `clone $obj`, even phpcbf with cakephp rules warns about *useless parentheses*

Reply:
But there's no space after `clone`. It should be `clone $throttle` not `clone$throttle`.

Reply:
see also failed travis build https://travis-ci.org/github/UseMuffin/Throttle/jobs/713659373

Reply:
i'll update the commit, but it's obviously not enforced by phpcs/phpcbf rules

Reply:
The fact the PHP doesn't throw a parse error when there is no space after `clone` is highly surprising to me.